automation/UNIX/test-lnt.sh

```diff
@@ -0,0 +1,36 @@
+# Test clang on UNIX using Visual Studio Team Services
+
+set -ue
+set -o pipefail
+set -x
```

Comment:
You probably want to remove this line unless you want bash to output all its debug information.